src/components/Signup/Form.test.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+
+import { render, waitFor, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { SignUpForm } from './Form';
+
+describe('singup Form:', () => {
+  const handleFormSubmit = jest.fn();
+
+  it('renders Sign up element', () => {
+    render(<SignUpForm handleFormSubmit={handleFormSubmit} />);
+
+    const inputNameElement = screen.getByPlaceholderText('Name');
+    expect(inputNameElement).toHaveAttribute('type', 'text');
+    expect(inputNameElement).toHaveValue('');
+
+    const inputEmailElement = screen.getByPlaceholderText('email');
+    expect(inputEmailElement).toHaveAttribute('type', 'email');
+    expect(inputEmailElement).toHaveValue('');
+
+    const inputPasswordElement = screen.getByPlaceholderText('password');
+    expect(inputPasswordElement).toHaveAttribute('type', 'password');
+    expect(inputPasswordElement).toHaveValue('');
+
+    const submitButtonElement = screen.getByRole('button', { name: 'Save' });
+    expect(submitButtonElement).toHaveAttribute('type', 'submit');
+
+    const resetButtonElement = screen.getByRole('button', { name: 'Cancel' });
+    expect(resetButtonElement).toHaveAttribute('type', 'reset');
+  });
+
+  it('submits values of form on submit', async () => {
```

Comment:
Make sure to write correct English 🧑‍🏫 

```suggestion
  it('submits the form's input values on submit', async () => {
```